rust/rope/src/spans.rs

```diff
@@ -189,6 +195,15 @@ impl<T: Clone + Default> Spans<T> {
             ix: 0,
         }
     }
+
+    /// iterates through all spans, but in backwards order
```

Comment:
Do you think it is necessary to find a appropriate new position for such a comment, since I've implemented `DoubleEndedIterator`? Or is `DoubleEndedIterator` clear enough about what to expect?